cmd/test/main.go

```diff
@@ -198,20 +204,29 @@ func New(ctx *TestCommandContext) *cobra.Command {
 				return err
 			}
 
-			testCommandFlags := TestCommandFlags{Output: outputFlag, K8sVersion: k8sVersion, IgnoreMissingSchemas: ignoreMissingSchemas, PolicyName: policy, SchemaLocations: schemaLocations, OnlyK8sFiles: onlyK8sFiles}
-			err = testCommandFlags.Validate()
+			localConfigContent, err := ctx.LocalConfig.GetLocalConfiguration()
 			if err != nil {
 				return err
 			}
 
-			localConfigContent, err := ctx.LocalConfig.GetLocalConfiguration()
+			prerunDataForEvaluation, statusCode, err := ctx.CliClient.RequestPrerunDataForEvaluation(localConfigContent.CliId)
+
+			if err != nil {
+				if statusCode != 404 {
```

Comment:
I think we need to add a comment here to explain this behavior. if someone is reading this it will be unclear what's the purpose of this exception